src/css/cssCastUnit.js

```diff
@@ -0,0 +1,60 @@
+define([
+	"../core",
+	"../var/rfxnum"
+], function( jQuery, rfxnum ) {
+
+function cssCastUnit( elem, prop, parts, tween ) {
+	var getCSS = tween ?
+			function() { return tween.cur(); } :
+			function() { return jQuery.css( elem, prop, "" ); },
+		target = getCSS(),
+		unit = parts && parts[ 3 ] || ( jQuery.cssNumber[ prop ] ? "" : "px" ),
+		// Starting value computation is required for potential unit mismatches
+		start = ( jQuery.cssNumber[ prop ] || unit !== "px" && +target ) &&
+			rfxnum.exec( jQuery.css( elem, prop ) ),
+		end,
```

Comment:
Declare this uninitialized variable first: http://contribute.jquery.org/style-guide/js/#assignments